app/queries/drug_stocks_query.rb

```diff
@@ -3,46 +3,60 @@ class DrugStocksQuery
 
   CACHE_VERSION = 1
 
-  def initialize(facilities:, for_end_of_month:, include_block_report: true)
```

Comment:
Are we biting the bullet and always including block-level info? Let's capture that in the PR description, it's useful for reviewers.